### PR TITLE
MISUV-3701_Fix_Welsh Fix Welsh Payment method

### PR DESCRIPTION
**Pull request by andriikondiuk**

### CODE REVIEW

File: app/views/RefundToTaxPayer.scala.html [hunk 1]
```diff
@@ -73,7 +73,9 @@
         <div class="govuk-summary-list__row">
             <dt class="govuk-summary-list__key">@messages("refund-to-taxpayer.tableHead.method")</dt>
             <dd class="govuk-summary-list__value">@{
-                repaymentHistoryModel.repaymentsViewerDetails.head.repaymentMethod
+                messages(s"refund-to-taxpayer.method-type-${
+                    repaymentHistoryModel.repaymentsViewerDetails.head.repaymentMethod.replace(" ", "-").toLowerCase
```
Comment (avelx) [inline app/views/RefundToTaxPayer.scala.html:77]: pls use headOption instead

File: test/views/RefundToTaxPayerViewSpec.scala [hunk 2]
```diff
@@ -90,13 +93,23 @@ class RefundToTaxPayerViewSpec extends ViewSpec with ImplicitDateFormatter {
   val testRepaymentHistoryModelRequestedAmountDiffersToRefundAmount: RepaymentHistoryModel =
     testRepaymentHistoryModel.copy(
       List(
-        testRepaymentHistoryModel.repaymentsViewerDetails.head.copy(amountApprovedforRepayment = Some(800.12), amountRequested = 345.5))
+        testRepaymentHistoryModel.repaymentsViewerDetails.head.copy(
```
Comment (avelx) [inline test/views/RefundToTaxPayerViewSpec.scala:96]: pls use headOption instead

File: test/views/RefundToTaxPayerViewSpec.scala [hunk 3]
```diff
@@ -90,13 +93,23 @@ class RefundToTaxPayerViewSpec extends ViewSpec with ImplicitDateFormatter {
...
+          repaymentMethod = RefundToTaxPayerMessages.tableValueMethodTypeCard
+        )
+      )
     )
 
   val testRepaymentHistoryModelRequestedMissingRefundAmount: RepaymentHistoryModel =
     testRepaymentHistoryModel.copy(
       List(
-        testRepaymentHistoryModel.repaymentsViewerDetails.head.copy(amountApprovedforRepayment = None, amountRequested = 345.5))
+        testRepaymentHistoryModel.repaymentsViewerDetails.head.copy(
```
Comment (avelx) [inline test/views/RefundToTaxPayerViewSpec.scala:107]: pls use headOption instead

**Review by avelx — CHANGES_REQUESTED** [review]
minor changes required

File: app/views/RefundToTaxPayer.scala.html (hunk 1 above)
Comment (andriikondiuk) [reply app/views/RefundToTaxPayer.scala.html]: Thank you. Done

**Review by avelx — APPROVED** [review]
looks good

**Pull request merged**